src/methods/artoo.methods.scrape.js

```diff
@@ -45,7 +45,8 @@
 
   // Scraping function after polymorphism has been taken care of
   function scrape(iterator, data, params, cb) {
-    var scraped = [],
+    var $ = artoo.$,
```

Comment:
Can I be extremely nitpicky and ask you to replace the remaining occurrences of `artoo.$` by `$` in the `scrape` function so only one name is used?